book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] I think it would be great if you make retals `attr_accessor` as well since also we need to set and get the rental not just get it since `attr_reader` it is just get the rental so kindly update it ⭐🚀✅📌🔥